packages/components/pie-modal/src/modal.scss

```diff
@@ -48,10 +49,23 @@
         // We only apply this when the modal is open,
         // otherwise it interferes with the native
         // `display: none;` on the dialog element.
-        display: flex;
-        flex-direction: column;
+        display: grid;
+    }
+
+    grid-template-areas:
+        'header'
+        'content';
+    grid-template-rows: min-content minmax(var(--modal-content-min-height), 1fr);
+
+    &[isfooterpinned] {
```

Comment:
Nit – Can you move this to be after the other base styling declarations? (so somewhere after line 84)

Not sure if it's just me but I like to have all the base styles together before hitting any modifiers (just makes it easier to scan the base styles)